bin/mail/2016-11-03-hoc-global-site-refresh/generate-teachers

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# 3 Categories (in order), represented by a category param:
+# 1. signup2016:
+#   - HocSignup2016
+# 2. past_organizer
+#   - HocHardwarePrizes2014
+#   - HocHardwarePrizes2015
+#   - HocSignup2014
+#   - HocSignup2015
+#   - HocSignup2016
+#   - HocSurvey2014
+#   - HocSurvey2015
+#   - CSEdWeekEvent2013
+# 3. teacher
+#   - Code Studio teachers
+#   - Petition (educators)
+#   - BringToSchool2013
+#   - ClassSubmission
+#   - DistrictPartnerSubmission
+#   - PLP interest form
+#   - Teacher interest form
+#   - School interest form
+#   - HelpUs2013
+#   - K5OnlineProfessionalDevelopmentPostSurvey
+#   - K5ProfessionalDevelopmentSurvey
+#   - ProfessionalDevelopmentWorkshop
+#   - ProfessionalDevelopmentWorkshopSignup
+#   - StudentNomination
+#   - TeacherNomination
+
+queries = {
+  signup2016: 'kind_s: HocSignup2016',
+
+  past_organizer: '
+    kind_s: HocHardwarePrizes2014 ||
+    kind_s: HocHardwarePrizes2015 ||
+    kind_s: HocSignup2014 ||
+    kind_s: HocSignup2015 ||
+    kind_s: HocSignup2016 ||
+    kind_s: HocSurvey2014 ||
+    kind_s: HocSurvey2015 ||
+    kind_s: CSEdWeekEvent2013
+  ',
+
+  teacher: '
+    (
+      kind_s:user && role_s:teacher
+    ) || (
+      kind_s:Petition && role_s: educator
```

Comment:
super nit: Inconsistent spacing (add spaces before `user`, `teacher`, and `Petition`).